features/support/paths.rb

```diff
@@ -20,6 +20,7 @@ def path_to(page_name)
     when /^the new donation page$/ then '/donations/new'
     when /^the first task edit page$/ then '/events/1/tasks/1/edit'
     when /^the add task page$/ then '/events/1/tasks/new'
+    when /^the tasks page$/ then '/events/1/tasks'
```

Comment:
step does not say anything about it only being for the first event.

Reply:
I tried to remove the @events assignment but then the task did not retain its event info correctly.

Reply:
also i am fixing merge conflicts now and then i will repush